.tx/config

```diff
@@ -36,38 +36,255 @@ source_lang  = en
 type         = YML
 minimum_perc = 0
 
+[o:energy-sparks:p:energy-sparks:r:en-yml]
+file_filter  = config/locales/<lang>/en.yml
+source_file  = config/locales/en.yml
+source_lang  = en
+type         = YML
+minimum_perc = 0
+
+[o:energy-sparks:p:energy-sparks:r:helpers-application_helper-yml]
+file_filter  = config/locales/<lang>/helpers/application_helper.yml
+source_file  = config/locales/helpers/application_helper.yml
+source_lang  = en
+type         = YML
+minimum_perc = 0
+
+[o:energy-sparks:p:energy-sparks:r:helpers-timeline_helper-yml]
+file_filter  = config/locales/<lang>/helpers/timeline_helper.yml
+source_file  = config/locales/helpers/timeline_helper.yml
+source_lang  = en
+type         = YML
+minimum_perc = 0
+
 [o:energy-sparks:p:energy-sparks:r:mailers-mailer-yml]
 file_filter  = config/locales/<lang>/mailers/mailer.yml
 source_file  = config/locales/mailers/mailer.yml
 source_lang  = en
 type         = YML
 minimum_perc = 0
 
```

Comment:
does there need to be a config for the `config/locales/analytics` dir?  only one file in there called `common.yml`

Reply:
You can't configure a directory, only specific files. So for now there's just the common.yml. There will be others as we extract text from the analytics.